spec/models/ems_refresh/save_inventory_spec.rb

```diff
@@ -161,24 +151,14 @@
         EmsRefresh.save_vms_inventory(@ems, data)
 
         vms = Vm.all
-        expect(vms.length).to eq(3)
```

Comment:
Similarly here:
```
(byebug) data.map { |v| [v[:ems_ref], v[:uid_ems]] }
[["vm-0000000000001", "56346b53-84d8-460a-9f8f-7fed3132c3a9"], ["vm-0000000000002", "ada1a6be-af10-4325-a198-25bb3357ed03"]]
(byebug) Vm.all.map { |v| [v[:ems_ref], v[:uid_ems]] }
[["vm-0000000000001", "56346b53-84d8-460a-9f8f-7fed3132c3a9"], ["vm-0000000000002", "56346b53-84d8-460a-9f8f-7fed3132c3a9"]]
```

So we're picking the existing VM with the same ems_ref and updating the uid_ems to match the new data